src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userservice = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User saveduser = userservice.findByEmail(email);
```

Comment:
yep, that the correct approach :+1: 
there is no way to remove the `user` variable from here and no need for that either, so it was just a misunderstanding, 
Next time feel free to tag the mentor and ask any questions regarding the review comments in personal chat on the platform :smiley: